### PR TITLE
Gameplay

### PR DESCRIPTION
**Pull request by dumitriucristian**
This is my version of gameplay. We could use this or something else. Please take a look and if you agree, merge with the master and develop branch and try to build it.

### CODE REVIEW

**Review by RobertPlaiasu — APPROVED** [review]
We use any javascript for the map ?

File: gameplay.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## Maps
+Given we have a map  of 100 x 100 squares
```
Comment (GrigoAleex) [inline gameplay.md:2]: I think that the 100x100 map is a little too big for the beginning. What do you think? What about an initial 10x10 map and then scale up as much as we want? I hope that we will talk more over the next meeting 😃!

**Review by GrigoAleex — APPROVED** [review]
I do not know where to take these ideas, but they are awesome! I hope we will have enough time for all of this 🙏!

File: gameplay.md (hunk 1 above)
Comment (dumitriucristian) [reply gameplay.md:2]: Great suggestion.

**Pull request merged**